UnityProject/Assets/Scripts/ScriptableObjects/RP/EmoteBehaviors/GiveSpeedStatusEffect.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Systems.StatusesAndEffects;
+using UnityEngine;
+
+namespace ScriptableObjects.RP.EmoteBehaviors
+{
+	public class GiveSpeedStatusEffect : IEmoteBehavior
```

Comment:
why an emote? Where are you calling this emote?

Reply:
`CoughAndGiveSpeed.asset`
The emote is getting called in `TobaccoInBody.asset` as its an reaction that triggers emotes.

Reply:
ah, fair